libs/eo/landing-page/shell/src/lib/eo-landing-page.integration.spec.ts

```diff
@@ -38,7 +45,21 @@ describe('EnergyOrigin landing page', () => {
       ],
     });
     await navigate('/');
+  });
 
+  it('displays the EnergyOrigin logo', async () => {
     expect(await findEnergyOriginLogo()).toBeInTheDocument();
   });
+
+  it('displays the EnergiNet logo', async () => {
+    expect(await findEnergiNetLogo()).toBeInTheDocument();
+  });
```

Comment:
issue: Always *Energinet*, never *EnergiNet*.
```suggestion
  it('displays the Energinet logo', async () => {
    expect(await findEnerginetLogo()).toBeInTheDocument();
  });
```